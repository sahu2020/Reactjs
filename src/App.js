import React from 'react';
import NavigationBar from "./Components/Navbar/Navbar";
import 'bootstrap/dist/css/bootstrap.min.css';
import Footer from "./Components/Footer/Footer";
import Card from "./Components/Cards/Card";
import './App.css';
import {Provider} from 'react-redux'
import { store } from './redux/Store';
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import Items from "./Components/Items/Items";
import MyCart from "./Components/MyCart/MyCart";



function App() {
  return (
    <div className="app-container">
      <Provider store={store}>
        <NavigationBar />
         <Router>
        <Switch>
          <Route path={"/"} exact component={Items} />
          <Route path={"/cart"} component={MyCart} />
        </Switch>
        </Router>
        <div
          style={{
            marginTop: "80px",
            display: "flex",
            justifyContent: "center",
            alignItems: "center",
          }}
        >
        </div>
        <Footer />
      </Provider>
    </div>
  );
}

export default App;
