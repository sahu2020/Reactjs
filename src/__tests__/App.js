import React from "react";
import {mount} from 'enzyme';
import App from '../App';
describe('App tests', () => {
   it('App component renders correctly', () => {
       const component = mount(<App/>);
       expect(component).toMatchSnapshot();
   });
});