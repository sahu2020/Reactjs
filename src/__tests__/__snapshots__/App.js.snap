// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests App component renders correctly 1`] = `ReactWrapper {}`;
